src/Compilers/Core/Portable/Symbols/ILocalSymbol.cs

```diff
@@ -64,5 +64,21 @@ public interface ILocalSymbol : ISymbol
         /// Returns true if the local variable is declared with fixed-pointer-initializer (in unsafe context).
         /// </summary>
         bool IsFixed { get; }
+
+        /// <summary>
+        /// Returns true if this local variable is declared as iteration variable
+        /// </summary>
+        bool IsForEach { get; }
+
+        /// <summary>
+        /// Returns true if the local variable is declared in resource-acquisition of a 'using statement';
+        /// otherwise false
+        /// </summary>
+        /// <example>
+        /// <code>
+        ///     using (var localVariable = new StreamReader("C:\\Temp\\MyFile.txt")) { ... } 
```

Comment:
>"C:\\Temp\\MyFile.txt" [](http://example.com/codeflow?start=60&length=22)

Perhaps use `path` in the example, rather than a platform-specific path.

Reply:
Changed: d2ccd3ece2e22cebc33c24b61af8e2043737045c
But the docstring is copied from `LocalSymbol` (it seems to me the other docstrings on `ILocalSymbol` are copied from `LocalSymbol`, as well, so I did the same for consistency):
https://github.com/dotnet/roslyn/blob/ace7a2b08bf38d9ac80cd41980a907ec9162dbe9/src/Compilers/CSharp/Portable/Symbols/LocalSymbol.cs#L224